node/pkg/logscribe/app.go

```diff
@@ -87,3 +148,125 @@ func bulkCopyLogs(ctx context.Context, logsChannel <-chan *[]api.LogInsertModel)
 		}
 	}
 }
+
+func ProcessLogs(ctx context.Context) map[string][]LogInsertModelWithIDWithCount {
+	processedLogs := make(map[string][]LogInsertModelWithIDWithCount)
+	logMap := make(map[string]map[string][]LogInsertModelWithID) // {"service": {hashedLog: []logs}}
```

Comment:
I don't see the point why the hash should be generated inside this function.
If hash is to be used as a key, why not just use the `LogInsertModelWithID` type as a key.

hashing being done again from `createGithubIssues` func

Reply:
I personally try to avoid this kind of nested map type, it makes the code less readable, and in most cases it could be avoided

Reply:
I see

Reply:
I have another question, it ID and Timestamps being utilized at the moment?
if not, I think the code could be more simplified

Reply:
even though timestamp is being used, it's just one of multiple timestamps that's being used right?

Reply:
I think timestamp won't be referenced that much for debugging unless we can see all the list of timestamps for same error. 

So if it's not used for hash generation nor showing up from github issue, I think it'd be okay to be removed